README.md

```diff
@@ -33,7 +33,7 @@ Follow these simple steps to install and set up our SSW Rules GPT project:
 4. Run the WebUI project, and a browser window will open to https://localhost:5002
    
 > **Warning**  
-> RulesGPT has been changed from port 5001 to 5002, if you are running Identity Server locally you **must** be running on port 5002.  
+> The RulesGPT frontend has been changed from port 5001 to 5002, if you are running Identity Server locally you **must** be running on port 5002.  
```

Comment:
```suggestion
> The RulesGPT frontend has been changed from port 5001 to 5002, if you are running Identity Server locally (port 5001) you **must** be running RulesGPT on port 5002.  
```

Reply:
@MattParkerDev IdentityServer runs on port 5003. 
RulesGPT was changed to 5002 because SSW Rewards runs on 5001 and it was likely to cause confusion in the Identity Server code as to what was supposed to run on 5001.

The reason you need to be running on 5002 with Identity Server is because it will block the request if it doesn't originate from 5002.

I agree with adding "running RulesGPT on port..." though, I'll make the change.